libs/wingc/src/jsify.rs

```diff
@@ -718,9 +718,9 @@ impl<'a> JSifier<'a> {
 	/// 	}
 	/// }
 	/// _toInflight() {
-	/// 	const b_client = this.b._toInflight();
+	/// 	const b_client = this._lift(b);
```

Comment:
I wonder if we'll at some point want `lift` to be some kind of method on the inflight host (not the resource). For example, a `cloud.Function` would be responsible for "lifting" a value into a JS string, and a `cloud.Workflow` could "lift" a value into some state machine language value or AST representation, etc.

(for future work)

Reply:
Interesting concept, but I think as long as out inflights are in a functions/imperitive language (and not a state machine) it makes sense the way it is now.

Reply:
yes, we'll get there! For now, let's say that the resource "lifts" the value into an inflight context.